src/features/commands/reportMod.js

```diff
@@ -0,0 +1,52 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const db = require('../../db');
+
+module.exports = {
+    type: 'slash',
+    data: new SlashCommandBuilder()
+        .setName('reportmod')
+        .setDescription('Manage reports')
+        .addSubcommand(command =>
+            command
+                .setName('block')
+                .setDescription('Block an user from using report commands')
+                .addUserOption(option =>
+                    option
+                        .setName('user')
+                        .setDescription('User to block')
+                        .setRequired(true)))
+        .addSubcommand(command =>
+            command
+                .setName('unblock')
+                .setDescription('Reallow an user to use report commands')
+                .addUserOption(option =>
+                    option
+                        .setName('user')
+                        .setDescription('User to block')
+                        .setRequired(true))),
+
+    async execute(client, interaction) {
+        switch(interaction.options.getSubcommand()) {
+            case 'block': {
+                const user = interaction.options.getUser('user');
+    
+                await db.saveBlockedReportUser(user.id, user.username);
+        
+                interaction.reply({
+                    content: `:exclamation: The user ${user} is now blocked from using report commands.`,
+                    ephemeral: true
+                })
+            } break;
+            case 'unblock': {
+                const user = interaction.options.getUser('user');
+    
+                await db.deleteBlockedReportUser(user.id);
+        
+                interaction.reply({
+                    content: `:white_check_mark: The user ${user} can now use report commands.`,
+                    ephemeral: true
+                })
+            } break;
+        }
+    }
+}
```

Comment:
We should have permission checks here, to make sure that everyone isn't able to use these commands.

Reply:
Yep I didn't bother trying to figure out how to check for permissions, but like in index.js, I think `message.member.permissions.serialize().KICK_MEMBERS` should be enough to check whether or not an user is a mod, or maybe eventually check if the user has access to the report channel

Reply:
> I think `message.member.permissions.serialize().KICK_MEMBERS` should be enough to check whether or not an user is a mod

Certainly more than enough, as for checking whether mod has access to the channel seems kind of pointless as they wouldn't be able to interact anyways if they can't access the channel.